src/bot/commands/weather.js

```diff
@@ -73,6 +73,10 @@ function getEmojiIcon(type) {
   }
 }
 
+function convertFToC(temperature) {
```

Comment:
I'd spell out fahrenheit and celcius

Reply:
Also since we're doing `XF / YC` in a bunch of places, it might be nice to do a 

```javascript
function temperature(fahrenheit) {
  const celcius = (fahrenheit - 32) * 5 / 9;
  return `${Math.round(fahrenheit)}°F/${Math.round(celcius)}°C`;
}
```